locales/fr.json

```diff
@@ -279,6 +291,10 @@
       "pass": "Les attributs ARIA sont utilisés correctement pour le rôle défini",
       "fail": "{{=it.data && it.data.length > 1 ? 'Attributs ARIA non autorisés' : 'Attribut ARIA non autorisé'}} : {{~it.data:value}} {{=value}}{{~}}"
     },
+    "aria-allowed-role": {
+      "pass": "Le rôle ARIA role est autorisé pour l’élément donné",
+      "fail": "{{=it.data && it.data.length > 1 ? 'Les rôles' : 'Le rôle'}} {{=it.data.join(', ')}} {{=it.data && it.data.length > 1 ? 'ne sont' : 'n’est'}} pas autorisé pour l’élément donné"
```

Comment:
Conjugation to correct:
`{{=it.data && it.data.length > 1 ? 'ne sont pas autorisés' : 'n’est pas autorisé'}}`
instead of:
`{{=it.data && it.data.length > 1 ? 'ne sont' : 'n’est'}} pas autorisé`